pkg/steampipeconfig/modconfig/workspace_profile.go

```diff
@@ -125,22 +192,42 @@ func (p *WorkspaceProfile) ConfigMap() map[string]interface{} {
 	res.SetStringItem(p.SnapshotLocation, constants.ArgSnapshotLocation)
 	res.SetStringItem(p.WorkspaceDatabase, constants.ArgWorkspaceDatabase)
 	res.SetIntItem(p.QueryTimeout, constants.ArgDatabaseQueryTimeout)
+	res.SetBoolItem(p.Watch, constants.ArgWatch)
+	res.SetIntItem(p.MaxParallel, constants.ArgMaxParallel)
+	res.SetStringSliceItem(stringToSlice(p.SearchPath, ","), constants.ArgSearchPath)
+	res.SetStringSliceItem(stringToSlice(p.SearchPathPrefix, ","), constants.ArgSearchPathPrefix)
+	res.SetBoolItem(p.Introspection, constants.ArgIntrospection)
+	res.SetBoolItem(p.Input, constants.ArgInput)
+	res.SetBoolItem(p.Progress, constants.ArgProgress)
+	res.SetStringItem(p.Theme, constants.ArgTheme)
+	res.SetBoolItem(p.Cache, constants.ArgCache)
+	res.SetIntItem(p.CacheTTL, constants.ArgCacheTtl)
 
-	// now add options
-	// build flat config map with order or precedence (low to high): general, terminal, connection
-	// this means if (for example) 'search-path' is set in both terminal and connection options,
-	// the value from connection options will have precedence
-	// however, we also store all values scoped by their options type, so we will store:
-	// 'database.search-path', 'terminal.search-path' AND 'search-path' (which will be equal to 'terminal.search-path')
-	if p.GeneralOptions != nil {
-		res.PopulateConfigMapForOptions(p.GeneralOptions)
+	if cmd.Name() == "query" && p.QueryOptions != nil {
```

Comment:
can we add consts for command names please

Reply:
:+1: